lib/mods_display/fields/publisher.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ModsDisplay
+  class Publisher < Field
+    def fields
+      return_fields = @values.map do |value|
+        publisher_value = Stanford::Mods::Imprint.new(value).send(:publisher_vals_str)
```

Comment:
If this isn't a public method on `Stanford::Mods::Imprint`, a dev might change it without thinking twice.

Reply:
@justinlittman https://github.com/sul-dlss/stanford-mods/pull/146